src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const sum = days * 40;
+
+  if (days < 3) {
+    return sum;
+  }
+
+  if (days < 7) {
+    return sum - 20;
+  }
+
+  return sum - (Math.floor(days / 7) * 50);
```

Comment:
According to the checklist, everything is correct - that is approved.
But I would not do calculations in return. It is better to make them earlier, and return a ready-made value. This makes the code easier to read.